docs/solidity-by-example.rst

```diff
@@ -327,18 +322,15 @@ activate themselves.
             // external contracts.
 
             // 1. Conditions
-            if (now <= auctionStart + biddingTime)
-                throw; // auction did not yet end
-            if (ended)
-                throw; // this function has already been called
+            require(now >= auctionStart + biddingTime); // auction did not yet end
```

Comment:
I know the type check would catch it if the precedence resolution would be broken, but I don't think it is bad practice having parentheses.